cppForSwig/Makefile

```diff
@@ -17,7 +17,14 @@ endif
 platform=$(shell uname)
 
 ifeq ($(shell uname), Darwin)
+# XCode 8 breaks when compiling for 10.7, XCode 7 and below does not
+XCODE_VERSION:=$(shell xcodebuild -version | awk '{ print $$2 }' | head -n1)
+ifeq ($(shell echo "${XCODE_VERSION} < 8" | bc), 0)
```

Comment:
This line ought to be version > 7. In addition, I'm getting the following error on 10.12.3.

```(standard_in) 1: parse error```